esphome/components/bme680_bsec/__init__.py

```diff
@@ -62,8 +62,10 @@ async def to_code(config):
         var.set_state_save_interval(config[CONF_STATE_SAVE_INTERVAL].total_milliseconds)
     )
 
-    if CORE.is_esp32:
-        # Although this component does not use SPI, the BSEC library requires the SPI library
+    # Although this component does not use SPI, the BSEC library requires the SPI library
+    if CORE.is_esp32 and CORE.using_arduino:
+        cg.add_library("SPI", None)
+    if CORE.is_esp8266:
         cg.add_library("SPI", None)
```

Comment:
This component only works on Arduino anyway, so let's just unconditionally add the SPI library:
```suggestion
    # Although this component does not use SPI, the BSEC library requires the SPI library
    cg.add_library("SPI", None)
```